src/include/cephd/libcephd.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CEPH_LIBCEPHD_H
+#define CEPH_LIBCEPHD_H
```

Comment:
@tchaikov thanks for the feedback. I'll go ahead and make the changes you suggested, and favor improvement over consistency.